src/desktopvirtualization/azext_desktopvirtualization/tests/latest/test_desktopvirtualization_scenario.py

```diff
@@ -0,0 +1,268 @@
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+#
+# Code generated by Microsoft (R) AutoRest Code Generator.
+# Changes may cause incorrect behavior and will be lost if the code is
+# regenerated.
+# --------------------------------------------------------------------------
+
+import os
+from azure.cli.testsdk import ScenarioTest
+from .. import try_manual, raise_if
+from azure.cli.testsdk import ResourceGroupPreparer
+
+
+TEST_DIR = os.path.abspath(os.path.join(os.path.abspath(__file__), '..'))
+
+
+@try_manual
+def setup(test, rg):
+    pass
+
+
+# EXAMPLE: /HostPools/put/HostPool_Create
+@try_manual
+def step__hostpools_put_hostpool_create(test, rg):
+    test.cmd('az desktopvirtualization hostpool create '
+             '--location "centralus" '
+             '--description "des1" '
+             '--friendly-name "friendly" '
+             '--host-pool-type "Pooled" '
+             '--load-balancer-type "BreadthFirst" '
+             '--max-session-limit 999999 '
+             '--personal-desktop-assignment-type "Automatic" '
+             '--registration-info expiration-time="2020-10-01T14:01:54.9571247Z" registration-token-operation="Update" '
+             ''
+             '--sso-context "KeyVaultPath" '
+             '--tags tag1="value1" tag2="value2" '
+             '--name "{hostPool1}" '
+             '--resource-group "{rg}"',
+             checks=[])
+
+
+# EXAMPLE: /HostPools/get/HostPool_Get
+@try_manual
+def step__hostpools_get_hostpool_get(test, rg):
+    test.cmd('az desktopvirtualization hostpool show '
+             '--name "{hostPool1}" '
+             '--resource-group "{rg}"',
+             checks=[])
+
+
+# EXAMPLE: /HostPools/get/HostPool_List
+@try_manual
+def step__hostpools_get_hostpool_list(test, rg):
+    test.cmd('az desktopvirtualization hostpool list '
+             '-g ""',
+             checks=[])
+
+
+# EXAMPLE: /HostPools/get/HostPool_ListByResourceGroup
+@try_manual
+def step__hostpools_get_hostpool_listbyresourcegroup(test, rg):
+    test.cmd('az desktopvirtualization hostpool list '
+             '--resource-group "{rg}"',
+             checks=[])
+
+
+# EXAMPLE: /HostPools/patch/HostPool_Update
+@try_manual
+def step__hostpools_patch_hostpool_update(test, rg):
+    test.cmd('az desktopvirtualization hostpool update '
+             '--description "des1" '
+             '--friendly-name "friendly" '
+             '--load-balancer-type "BreadthFirst" '
+             '--max-session-limit 999999 '
+             '--personal-desktop-assignment-type "Automatic" '
+             '--registration-info expiration-time="2020-10-01T15:01:54.9571247Z" registration-token-operation="Update" '
+             ''
+             '--sso-context "KeyVaultPath" '
+             '--tags "{{\\"tag1\\":\\"value1\\",\\"tag2\\":\\"value2\\"}}" '
+             '--name "{hostPool1}" '
+             '--resource-group "{rg}"',
+             checks=[])
```

Comment:
Could you add some checks to the results of these tests? Especially the verification of tag results~